src/components/ContinuationIn/SummaryBusinessHomeJurisdiction.vue

```diff
@@ -372,18 +405,43 @@ export default class SummaryBusinessHomeJurisdiction extends Mixins(DateMixin) {
   get expiryDate (): string {
     return this.yyyyMmDdToPacificDate(this.getContinuationAuthorization?.expiryDate, true, false)
   }
+
+  get affidavitDocument (): DocumentIF {
+    return {
+      title: 'Director Affidavit',
+      filename: this.getExistingBusinessInfo.affidavitFileName,
+      // link: this.getExistingBusinessInfo.affidavitFileUrl
+      link: 'https://minio-dev.apps.silver.devops.gov.bc.ca/businesses/36cfc3bd-dd76-4e6c-8234-521e5df09afb.pdf'
+    }
+  }
```

Comment:
I added this getter, and `authorizationDocument()` below, to convert from "our" file structure to the Document IF as used in other UIs. This allows for consistent fetch code across UIs.

Reply:
Obsolete comment.